Applications/spire/include/spire/book_view/technicals_panel.hpp

```diff
@@ -14,6 +14,40 @@ namespace Spire {
       //! Constructs a TechnicalsPanel.
       TechnicalsPanel(QWidget* parent = nullptr);
 
+      // Sets the text of the close label.
+      /*
+        \param close The new string to display.
+      */
+      void set_close(const QString& close);
```

Comment:
Instead of having all these setters, the TechnicalsPanel should take a model and update itself based on the model.